pkg/arvo/sys/vane/eyre.hoon

```diff
@@ -53,6 +56,10 @@
       $:  %gall
           gift:gall
           ::  $>(%unto gift:gall)
+          ::
+      ==
+      $:  %clay
+          gift:clay
```

Comment:
I think this should be `$>(%writ gift:clay)` -- using more specific types has a useful documentary effect, if nothing else.

Reply:
I added that, but then all HTTP events crashed. Unsure how to proceed, so I commented it out next to the type as it's done in the `%gall` option directly above it.

Reply:
The `$%` in `$sign` needs to be wrapped in an appropriate `$~`, then you can use `$>` for all these entries.

Reply:
And `$note` should get the same treatment, it only happens to work currently due to bunt order.